gprofiler/main.py

```diff
@@ -679,7 +688,7 @@ def parse_cmd_args() -> configargparse.Namespace:
         " given multiple times will append pids to one list",
     )
     parser.add_argument(
-        "--max-processes",
+        "--max-processes-runtime-profiler",
```

Comment:
The documentation and test cases are still using the "--max-processes" argument. We can keep both by doing something like:
```suggestion
        "--max-processes",
        "--max-processes-runtime-profiler",
```